sdk/attestation/azure-security-attestation/src/test/java/com/azure/security/attestation/AttestationPolicyManagementTests.java

```diff
@@ -0,0 +1,245 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.security.attestation;
+
+import com.azure.core.http.HttpClient;
+import com.azure.core.util.Configuration;
+import com.azure.security.attestation.models.AttestationCertificateManagementBody;
+import com.azure.security.attestation.models.JsonWebKey;
+import com.azure.security.attestation.models.PolicyCertificatesModifyResponse;
+import com.azure.security.attestation.models.PolicyCertificatesResponse;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.nimbusds.jose.*;
```

Comment:
We have a checkstyle for not using * imports. In intelliJ, we have to set the imports to 99.

![image](https://user-images.githubusercontent.com/10136526/105321015-c0484980-5b7b-11eb-9d74-1b6987c98855.png)

Reply:
Strangely, this appears to pass CheckStyle, but easy to fix.